dev-docs/workflows/build-develop-deploy.md

```diff
@@ -7,35 +7,21 @@ Prerequisites:
 * 20GB (minimum), better 40 GB disk space (required if you want to cross compile for all platforms)
 * [Latest version of Go](https://go.dev/doc/install).
 * Unless you use Nix / NixOS: [Bazelisk installed as `bazel` in your path](https://github.com/bazelbuild/bazelisk/releases).
-* We recommend Nix installed via [determinate systems installer](https://github.com/DeterminateSystems/nix-installer) (or NixOS as host system).
+* We require Nix installed via [determinate systems installer](https://github.com/DeterminateSystems/nix-installer) (or NixOS as host system).
```

Comment:
Hm, we required Nix installed, but should work with any installation, shouldn't it? So maybe separate requirement and recommendation here.